src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```
@@ -1,146 +1,164 @@
+  useEffect(() => {
+    fetchCategories({ page, limit: categoriesPerPage })
+      .then(() => {
+        setCategoriesLoaded(true)
+      })
+      .catch((error) => {
+        console.error('Error category download:', error)
+      })
+  }, [page, fetchCategories, setCategoriesLoaded])
+
+  useEffect(() => {
+    if (categories && categories.data && categories.data.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)